tests/unit/test_text.py

```diff
@@ -28,24 +28,123 @@
 @pytest.mark.parametrize("data", valid_string_data)
 def test_is_not_whitespace__with_valid_data__does_nothing(data):
     # Arrange & Act & Assert
-    _ = Text.is_not_whitespace(data)
+    Text.is_not_whitespace(data)
 
 
 @pytest.mark.parametrize("data", invalid_string_data)
-def test_is_not_whitespace__with_invalid_data__rises_argument_exception(data):
+def test_is_not_whitespace__with_invalid_data__raises_argument_exception(data):
     # Arrange & Act & Assert
     with pytest.raises(ArgumentException):
-        _ = Text.is_not_whitespace(data)
+        Text.is_not_whitespace(data)
 
 
 @pytest.mark.parametrize("data", valid_string_or_list_data)
 def test_is_not_empty__with_valid_data__does_nothing(data):
     # Arrange & Act & Assert
-    _ = Text.is_not_empty(data)
+    Text.is_not_empty(data)
 
 
 @pytest.mark.parametrize("data", invalid_string_or_list_data)
-def test_is_not_empty__with_invalid_data__rises_argument_exception(data):
+def test_is_not_empty__with_invalid_data__raises_argument_exception(data):
     # Arrange & Act & Assert
     with pytest.raises(ArgumentException):
-        _ = Text.is_not_empty(data)
+        Text.is_not_empty(data)
+
+
+def test_is_alphanumeric__with_non_alphanumeric_data__raises_argument_exception():
+    # Arrange & Act & Assert
+    with pytest.raises(ArgumentException):
+        Text.is_alphanumeric("*")

```

Comment:
It seems that you are suggesting that we should create tests for cpython's `is_alphanumeric` method. I guess the only thing our method is doing is raising an exception if the `is_alphanumeric` method returns `False`, which means that our unit test should only be validating whether an exception is raised or not. Maybe I should mock the behavior of cpython's `is_alpanumeric` method to return `False` instead of providing an example where it returns `False`?

Reply:
Already discussed this offline ✔️